src/client/utils/sentry-server.ts

```diff
@@ -1,5 +1,5 @@
 const sentryDsn = process.env.SENTRY_DSN
-const sentryEnvironment = process.env.SENTRY_ENVIRONMENT || 'dev'
+const sentryEnvironment = process.env.SENTRY_ENVIRONMENT
```

Comment:
Should we add these to the `.env.example` ?

Reply:
That's a good idea but I'm not going to do that now since that's not really in the scope of the PR I would argue. But yes, I totally agree!